shared/s3_bucket/main.tf

```diff
@@ -13,6 +13,11 @@ variable "acl" {
   default     = "public-read"
 }
 
+variable "force_public" {
```

Comment:
Is this not recommended, since we don't want the S3 URI's to be directly accessible, except for from `rogue` domain serving them?

Reply:
👹Totally agree. 💯